src/OpenFOAM/matrices/lduMatrix/solvers/PCGCache/PCGCache.H

```diff
@@ -16,10 +16,12 @@ class PCGCache
     static PtrList<scalargpuField> wTCache;
     static PtrList<scalargpuField> rTCache;
     
+    /*
```

Comment:
I would not want to leave this code commented here like this, since future readers will not have to context to know why it is commented and is it safe to delete.
I would prefer to add another commit that explicitly removes this code and code in the implementation, this way it will remain in the history hopefully with more context.